libs/sdk-ui-charts/api/sdk-ui-charts.api.md

```diff
@@ -524,6 +529,14 @@ export interface IDonutChartBucketProps {
 export interface IDonutChartProps extends IBucketChartProps, IDonutChartBucketProps {
 }
 
+// @beta
+export interface IForecast {
+    confidence: "70" | "75" | "80" | "85" | "90" | "95" | number;
```

Comment:
Does it need to be string numbers? I would either type it as number, or union of allowed numbers.

Reply:
This is special case to have better typecheck in config panel. From config panel we can use only these string values, in API of chart config we can use also numbers. The logic is try to parse nubmer from string, normalize it and convert to 0 - 1 finally. I can remove it, but it stop completion for config panels. We can keep there only number and string.